src/pages/careers/locations/index.tsx

```diff
@@ -199,38 +204,63 @@ const Locations = () => {
             />
             <Hero />
             <StyledSectionContainer direction="column" padding="80px 16px">
-                <Header
-                    as="h2"
-                    align="center"
-                    padding="0 0 35px"
-                    size={'var(--text-size-header-1)'}
-                >
-                    Explore our locations
-                </Header>
-                <CssGrid
-                    columns="repeat(3, 384px)"
-                    row_gap="40px"
-                    column_gap="24px"
-                    laptop_columns="repeat(3, minmax(280px, 384px))"
-                    laptop_margin="0 16px"
-                    tablet_columns="repeat(2, 1fr)"
-                    mobile_columns="minmax(300px, 384px)"
-                    mobile_row_gap="37px"
-                    style={{ marginTop: '0rem', justifyContent: 'center' }}
-                    margin="0 16px"
+                <StyledTabs
+                    tab_list={[
+                        'all',
+                        'europe',
+                        'asia',
+                        'middle_east',
+                        'eastern_europe',
+                        'africa',
+                        'latam',
+                    ]}
+                    jc_tablet="start"
+                    jc_mobileL="start"
+                    mobile_font_size={16}
+                    line_divider_length="unset"
                 >
-                    {all_offices.map((office) => (
-                        <CountryCard
-                            key={office.name}
-                            Icon={office.flagIcon}
-                            img_data={images[office.thumbnail]}
-                            country_name={office.country}
-                            city_name={office.display_name}
-                            link={office.link}
-                            img_alt={office.img_alt}
-                        />
-                    ))}
-                </CssGrid>
+                    {Object.keys(continents).map((key) => {
+                        return (
+                            <Tabs.Panel
+                                label={key
+                                    .split('_')
+                                    .filter((name) => name.length > 0)
+                                    .map((name) => name.charAt(0).toUpperCase() + name.slice(1))
```

Comment:
Create a function for this calculation, so it's easy to understand what's happening.